content/docs/deploy/enterprise/configure-terraform.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Configure with Terraform
+description: Learn how to manage your Pomerium Enterprise configuration using Terraform, including authentication setup, resource management, and deployment examples.
+keywords:
+  [
+    Pomerium Enterprise,
+    Terraform,
+    IaC,
+    infrastructure as code,
+    configuration management,
+    service accounts,
+    provider configuration,
+  ]
+---
+
+# Configure with Terraform
+
+Pomerium Enterprise can be configured and managed using Terraform through our official provider. This enables you to manage your Pomerium Enterprise resources as infrastructure as code, making it easier to version, review, and automate your configuration changes.
```

Comment:
consider adding a link to the provider on the Terraform registry.
```suggestion
Pomerium Enterprise can be configured and managed using Terraform through our [official provider](https://registry.terraform.io/providers/pomerium/pomerium/latest/docs). This enables you to manage your Pomerium Enterprise resources as infrastructure as code, making it easier to version, review, and automate your configuration changes.
```

Reply:
It might be obvious, but consider a note to avoid using the console UI or API directly for configuration if using the Terraform provider.